python/looker_sdk/rtl/requests_transport.py

```diff
@@ -104,6 +102,14 @@ def request(
 
         return ret
 
+    def make_path(
+        self, path: str, authenticator: Optional[Callable[[], Dict[str, str]]] = None
+    ) -> str:
+        base = self.api_path if authenticator else self.settings.base_url
```

Comment:
I'm pretty sure we don't want to use the presence/absence of `authenticator` to influence the url at all.

Reply:
My intent was that the authenticator is used as a flag to determine whether api_path was used as the base for relative paths, or whether base_url was. The thinking was, if the call needed to be authenticated, it's an API call.

Reply:
I could imagine someone wanting to provide a random `authenticator` callback to some fully qualified path they've set.

On a related, higher level note: the python transport interface is pretty limited. The return is just `bytes` (following @dthorpe's suggestion about leaving the response alone and letting calling code do what it wants to it). The serialization layer is separate (and only called in `api_methods.py`). So I actually don't think our transport is very helpful. If I were an app developer, I'd just import `requests` directly and have a lot easier time of it thank using our transport :-)

Reply:
Fully qualified path + authenticator should still work. Good point about the python transport interface limitations. We'll need to discuss that disparity with the other SDKs, because the other SDKs do have the ability to return strongly typed responses back from a given request.